src/app/providers/router/routeConfig/appRouteConfig.tsx

```diff
@@ -0,0 +1,53 @@
+import { FeedbackPage } from 'pages/feedback';
+import { AppRoutesProps } from './types';
+import { getFeedback, getQuestions, getMain, getNotFound, getResult, getCompletedSurvey } from './routes';
+import { NotFoundPage } from 'pages/notFound';
+import { MainPage } from 'pages/main';
+import { ResultPage } from 'pages/result';
+import { QuestionsPage } from 'pages/questions';
+import { CompletedSurveyPage } from 'pages/CompletedSurvey';
+
+export enum AppRoutes {
+  FEEDBACK = 'feedback',
+  QUESTIONS = 'questions',
+  RESULT = 'result',
+  COMPLETED_SURVEY = 'completed_survey',
+  MAIN = 'main',
+  NOT_FOUND = 'not_found',
```

Comment:
```suggestion
  COMPLETED_SURVEY = 'completed-survey',
  MAIN = 'main',
  NOT_FOUND = 'not-found',
```

это необязательно, предложение просто